app/components/market/submit/parameter-input.component.ts

```diff
@@ -45,7 +52,32 @@ export class ParameterInputComponent implements ControlValueAccessor, OnDestroy
     }
 
     public validate() {
-        return null;
+        const valid = this.parameterValue.valid;
+        if (valid) {
+            return null;
+        } else {
+            let messageText = `unknown error`;
```

Comment:
Nitpick: I noticed below you are using "" instead of ``. Any reason in particular? 

Example:
```ts
return "Click to submit form";
```

Reply:
should actually use `` only if using interpolation. Use "" otherwise